website/addons/dryad/views.py

```diff
@@ -0,0 +1,225 @@
+from pyDryad import Dryad_DataOne, Dryad_SOLR
```

Comment:
Not a blocker by any means, but the pep8 guidelines recommend no underscores in class names 

https://www.python.org/dev/peps/pep-0008/#class-names